### PR TITLE
tests and lint

### PR DESCRIPTION
**Pull request by moshfeu**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.) (I did run `npm run lint package-name`)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/expo/tslint.json [hunk 1]
```diff
@@ -2,6 +2,8 @@
     "extends": "dtslint/dt.json",
     "rules": {
         "void-return": false,
-        "max-line-length": false
+        "max-line-length": false,
+        "file-name-casing": false,
```
Comment (andy-ms) [inline types/expo/tslint.json]: Why was it necessary to disable this?

Comment (moshfeu) [reply types/expo/tslint.json]: Because the name of the of the tests file is `expo-tests.tsx` which fails the lint.

Comment (andy-ms) [reply types/expo/tslint.json]: This didn't fail before though, and the file name hasn't changed. This rule isn't in tslint [latest](https://github.com/palantir/tslint/blob/master/src/configs/latest.ts) so I suspect you are somehow using a different config than this one. Try removing the tslint.json changes and seeing if travis fails.

**Comment by moshfeu** [comment]
@andy-ms  Strange. I'm not sure why it's failing locally but I remove these changes

**Comment by andy-ms** [comment]
Thanks! Maybe you just need to delete `node_modules` and reinstall.

**Comment by moshfeu** [comment]
> Thanks! Maybe you just need to delete `node_modules` and reinstall.

Apparently, I installed dtslint globally probably with a different config. I uninstalled it globally and installed locally and now it's working.

Thanks for the feedback!

**Pull request merged**